### PR TITLE
gen/tests: Improve TLS configuration parameters tests

### PR DESCRIPTION
**Pull request by mhrabovcin**
## High-level description

Improves unit tests for TLS configuration parameters in `gen` package.

## Corresponding DC/OS tickets (obligatory)

These DC/OS JIRA ticket(s) must be updated (ideally closed) in the moment this PR lands:

  - [DCOS_OSS-2039](https://jira.mesosphere.com/browse/DCOS_OSS-2039) Address review comments from DCOS-19976

## Checklist for all PRs

  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain here:
  - [ ] Read the [DC/OS contributing guidelines](https://github.com/dcos/dcos/blob/master/contributing.md)
  - [ ] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

**Comment by mhrabovcin** [comment]
@mesosphere-mergebot label Ready For Review

**Comment by mhrabovcin** [comment]
@jgehrcke @vespian this one is ready for review.

File: gen/tests/test_adminrouter_tls_conf.py [hunk 1]
```diff
@@ -137,27 +167,39 @@ def supported_ssl_ciphers_agent(
 
     def test_cipher_agent_default(self):
         """
-        The config variable adminrouter_external_cipher_string should not impact internal traffic.
+        Admin Router Agent has comes with default ssl_ciphers configuration.
```
Comment (vespian) [inline gen/tests/test_adminrouter_tls_conf.py]: `s/has comes/comes/`
`s/with default/with the default/`

**Comment by mhrabovcin** [comment]
@mesosphere-mergebot bump-ee

**Comment by mhrabovcin** [comment]
@mesosphere-mergebot bump-ee

**Comment by timaa2k** [comment]
Reviewed and would approve, but need to wait for write access to dcos/dcos.

**Comment by mhrabovcin** [comment]
@mesosphere-mergebot ship_it_label_status_check

**Pull request merged**